src/components/10-atoms/button/index.js

```diff
@@ -76,11 +75,12 @@ class AXAButton extends LitElement {
         @click="${this.onClick}"
       >
         <div class="a-button__flex-wrapper">
-          <slot></slot>
-          ${icon &&
-            html`
-              <axa-icon class="a-button__icon" icon="${icon}"></axa-icon>
-            `}
+          <slot></slot> ${
+            icon &&
```

Comment:
Attention! This writes "false" to the DOM if icon is falsy.

Reply:
no changes here from my side, was prettier